src/GeositeFramework/js/Plugin.js

```diff
@@ -617,10 +621,11 @@ require(['use!Geosite',
             // If the plugin is not set up for a print modal,
             // resolve any pending pre-print map operations
             if (!pluginObject.usePrintModal) {
-                modalConfirmDeferred.resolve();
-                postModalDeferred.resolve();
-                mapReadyDeferred.resolve();
-                return mapReadyDeferred;
+                window.setTimeout(function() {
```

Comment:
What does the timeout help with here? Maybe add a comment?

Reply:
Good call. Will do.